examples/job/ml-train-model-predict-properties.ipynb

```diff
@@ -1,17 +1,19 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    from examples.utils.generic import dataframe_to_html, copy_bank_workflow_by_system_name, wait_for_jobs_to_finish, get_property_by_subworkow_and_unit_indicies, display_JSON
</pre><p> Some of these functions seem to be generic and useful enough that we could move them to the API client package. What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Good idea! I support moving generally useful functions to the client package, which will simplify this repo.